src/public/PermitInfo.1.json

```diff
@@ -740,8 +740,7 @@
     "name": "Nord Token"
   },
   "0x6f40d4a6237c257fff2db00fa0510deeecd303eb": {
-    "type": "eip-2612",
-    "version": "1",
+    "type": "unsupported",
```

Comment:
I guess this is related to https://cowservices.slack.com/archives/C0361CDG8GP/p1739786487403269, but I didn't get why it was detected as 2612 token, and manually changed later to unsupported

Reply:
It's possible their version was updated and/or some other change in their token implementation changed for which the permit script is not able to handle it.